added excerpt (#17)
publiccodenet/blog: _posts/2019-12-11-notes-from-community-call-26-september-2019.md

date: 2019-12-11
author: Jan Ainali
type: blogpost

> excerpt: This community call focused on archival criteria in the Standard for Public Code

Reviewer: ```suggestion
excerpt: We discussed archival criteria in the Standard for Public Code.
```
